testbed/compositions/graphsync.toml

```diff
@@ -16,23 +16,20 @@ instances = { count = 2 }
 
 [groups.build]
 
-
-
-
 [groups.run]
 [groups.run.test_params]
-input_data = "files"
-data_dir = "../extra/test-datasets"
+input_data = "random"
+file_size = "2000000000"
```

Comment:
Generating all this data may be really slow, what about using real data?

Reply:
@adlrocha @hannahhoward 

I agree with Alfonso here and am putting code to use real files on disk.

One more interesting thing I realised today is that Random File Generator uses an in-memory buffer: 

```
// RandReader generates random data from seed.
func SeededRandReader(len int, seed int64) io.Reader {
	randReader := rand.New(rand.NewSource(seed))
	data := make([]byte, len)
	randReader.Read(data)
	return bytes.NewReader(data)
}
```

and so if we use this, we don't actually do any disk IO on the send side of the transfer.

Opposed to this, when we give a file on disk to the tests, it will cause disk I/O and will be more in-line with GS using Badger. So, we should ONLY use pre-generated files going ahead for our benchmarks.